src/components/ReportActionItem/ReportActionItemImages.tsx

```diff
@@ -69,21 +71,33 @@ function ReportActionItemImages({images, size, total, isHovered = false}: Report
                     // Show a border to separate multiple images. Shown to the right for each except the last.
                     const shouldShowBorder = shownImages.length > 1 && index < shownImages.length - 1;
                     const borderStyle = shouldShowBorder ? styles.reportActionItemImageBorder : {};
+                    const isDistanceRequest = transaction && TransactionUtils.isDistanceRequest(transaction);
+                    const hasPendingWaypoints = transaction && TransactionUtils.isFetchingWaypointsFromServer(transaction);
+                    const showMapAsImage = isDistanceRequest && hasPendingWaypoints;
                     return (
                         <View
                             key={`${index}-${image}`}
                             style={[styles.reportActionItemImage, borderStyle, hoverStyle]}
                         >
-                            <ReportActionItemImage
-                                thumbnail={thumbnail}
-                                fileExtension={fileExtension}
-                                image={image}
-                                isLocalFile={isLocalFile}
-                                filename={filename}
-                                transaction={transaction}
-                                isThumbnail={isThumbnail}
-                                isSingleImage={numberOfShownImages === 1}
-                            />
+                            {showMapAsImage ? (
+                                <View style={[styles.w100, styles.h100]}>
```

Comment:
I think this is where some of those design bugs are coming from that @dannymcclain called out. We should be using our `ReportActionItemImage` component for all cases, we shouldn't be making some kind of exception here. 

@grgia do you mind taking a look at this since you helped write all of the preview component styles? Thanks!

Reply:
The main idea of this issue was basically to use Map view instead of `ReportActionItemImage ` to be consistent with money request preview and money request view so what we can do is tweek the style to make it same as before and I did that and please check it out

Reply:
Hmm I am not following. I think the idea is that all preview thumbnails for the report previews or expense previews always use the `ReportActionItemImage` component. Why can't we extend that component for this case?